extra/mender-client-docker-addons/Dockerfile

```diff
@@ -57,7 +58,8 @@ RUN mkdir -p /mender-install/var/lib/mender && echo device_type=generic-x86_64 >
 
 FROM ubuntu:22.04
 
-RUN mkdir -p /run/dbus && apt-get update && apt-get install -y liblzma5 dbus openssh-server
+RUN mkdir -p /run/dbus && apt-get update && apt-get install -y \
+    liblzma5 dbus openssh-server liblmdb0 libarchive13 libboost-log1.74.0
```

Comment:
Can we use just `libboost-log`? It will probably break on OS upgrade.

Reply:
`libboost-log` is not a package unfortunately :/